notes/stv/20210524-test-deploy-02.txt

```diff
@@ -0,0 +1,358 @@
+	changed: [worker02] => {"changed": true, "msg": "", "rc": 0, "results": ["Installed: kernel-5.6.13-100.fc30.x86_64", "Installed: kernel-core-5.6.13-100.fc30.x86_64", "Installed: kernel-modules-5.6.13-100.fc30.x86_64", "Installed: linux-firmware-20200421-107.fc30.noarch"]}
+	changed: [worker01] => {"changed": true, "msg": "", "rc": 0, "results": ["Installed: kernel-5.6.13-100.fc30.x86_64", "Installed: kernel-core-5.6.13-100.fc30.x86_64", "Installed: kernel-modules-5.6.13-100.fc30.x86_64", "Installed: linux-firmware-20200421-107.fc30.noarch"]}
+	changed: [master01] => {"changed": true, "msg": "", "rc": 0, "results": ["Installed: kernel-5.6.13-100.fc30.x86_64", "Installed: kernel-core-5.6.13-100.fc30.x86_64", "Installed: kernel-modules-5.6.13-100.fc30.x86_64", "Installed: linux-firmware-20200421-107.fc30.noarch"]}
+	changed: [monitor] => {"changed": true, "msg": "", "rc": 0, "results": ["Installed: kernel-5.6.13-100.fc30.x86_64", "Installed: kernel-core-5.6.13-100.fc30.x86_64", "Installed: kernel-modules-5.6.13-100.fc30.x86_64", "Installed: linux-firmware-20200421-107.fc30.noarch"]}
+	changed: [worker03] => {"changed": true, "msg": "", "rc": 0, "results": ["Installed: kernel-5.6.13-100.fc30.x86_64", "Installed: kernel-core-5.6.13-100.fc30.x86_64", "Installed: kernel-modules-5.6.13-100.fc30.x86_64", "Installed: linux-firmware-20200421-107.fc30.noarch"]}
+	fatal: [worker04]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ", "unreachable": true}
+
+...
+
+	PLAY RECAP 
+        ***************************************************************************************************************************************************************************
+	localhost                  : ok=84   changed=67   unreachable=0    failed=0    skipped=5    rescued=0    ignored=0   
+	master01                   : ok=85   changed=54   unreachable=0    failed=0    skipped=6    rescued=0    ignored=0   
+	monitor                    : ok=46   changed=35   unreachable=0    failed=0    skipped=1    rescued=0    ignored=0   
+	worker01                   : ok=91   changed=57   unreachable=0    failed=0    skipped=5    rescued=0    ignored=0   
+	worker02                   : ok=91   changed=57   unreachable=0    failed=0    skipped=5    rescued=0    ignored=0   
+	worker03                   : ok=91   changed=57   unreachable=0    failed=0    skipped=5    rescued=0    ignored=0   
+	worker04                   : ok=4    changed=2    unreachable=1    failed=0    skipped=0    rescued=0    ignored=0   
+	zeppelin                   : ok=98   changed=69   unreachable=0    failed=0    skipped=4    rescued=0    ignored=0   
+
+
+	real	82m8.394s
+	user	16m40.425s
+	sys	2m42.363s
+
+        # worker04 failed
+        # Not sure what went wrong, but after the step: /deployments/hadoop-yarn/ansible/04-update-fedora.yml , worker04 is no longer part of the deploy  
```

Comment:
The update-fedora script refreshes the dnf cache. If that times out, then the Ansible task fails. Once one step fails, that host is skipped for the rest of the `create-all` playbook. 
Issue #423 adds a retry loop to address this